scio-test/src/test/scala/com/spotify/scio/coders/CoderTest.scala

```diff
@@ -568,6 +568,13 @@ final class CoderTest extends AnyFlatSpec with Matchers {
     Recursive(1, Option(Recursive(2, None))) coderShould roundtrip()
   }
 
+  it should "#2644 throw a verifyDeterministic exception for Set" in {
```

Comment:
Small detail, the description of the test is inexact: `verifyDeterministic` is the name of the function, not the name of the exception